actions/helm_chart_e2e.go

```diff
@@ -0,0 +1,130 @@
+package actions
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/codegangsta/cli"
+	"github.com/google/go-github/github"
+)
+
+const (
+	// RepoFlag represents the '-repo' flag
+	RepoFlag = "repo"
+	// RefFlag represents the '-ref' flag (for specifying a SHA, branch or tag)
+	RefFlag = "ref"
+)
+
+// HelmChartE2E is the cli handler for generating a release helm chart for deis-e2e
+func HelmChartE2E(ghClient *github.Client) func(*cli.Context) {
+	return func(c *cli.Context) {
+		chartDir := "workflow-dev-e2e"
+		downloadFiles(ghClient, c, chartDir, []string{"README.md", "Chart.yaml"})
+
+		err := updateFilesWithRelease(chartDir)
+		if err != nil {
```

Comment:
can compress this and the above line with `if err := updateFilesWithRelease(...); err != nil {`